etc/no-region-apache.conf

```diff
@@ -0,0 +1,208 @@
+KeepAliveTimeout 75
+
+
+# The socket directory must be readable by the daemon process user
+WSGISocketPrefix /var/run/wsgi
+WSGIDaemonProcess encoded user=encoded group=encoded processes=6 threads=1 display-name=encoded-app
+# No need for embedded interpreters
+WSGIRestrictEmbedded On
+# Pass the authorization header so basic auth works
+WSGIPassAuthorization On
+
+# Indexer. Configure first to avoid catchall '/'
+WSGIDaemonProcess encoded-indexer user=encoded group=encoded processes=1 threads=1 display-name=encoded-indexer
+WSGIScriptAlias /_indexer /srv/encoded/parts/production-indexer/wsgi process-group=encoded-indexer application-group=%{GLOBAL}
+
+# visindexer. Configure first to avoid catchall '/'
+WSGIDaemonProcess encoded-visindexer user=encoded group=encoded processes=1 threads=1 display-name=encoded-visindexer
+WSGIScriptAlias /_visindexer /srv/encoded/parts/production-visindexer/wsgi process-group=encoded-indexer application-group=%{GLOBAL}
+
+# https://github.com/GrahamDumpleton/mod_wsgi/issues/2
+SetEnvIf Request_Method HEAD X_REQUEST_METHOD=HEAD
+
+LogFormat "%v:%p %h %l %u %t \"%r\" %>s %O \"%{Referer}i\" \"%{User-Agent}i\" %{X-Stats}o&server_time=%D" vhost_combined_stats
+
+<Directory /srv/encoded/parts/production-indexer>
+    Order deny,allow
+    Allow from all
+    <IfModule access_compat_module>
+        Require all granted
+    </IfModule>
+</Directory>
+
+<Directory /srv/encoded/parts/production-visindexer>
+    Order deny,allow
+    Allow from all
+    <IfModule access_compat_module>
+        Require all granted
+    </IfModule>
+</Directory>
+
+<Directory /srv/encoded/parts/production-regionindexer>
```

Comment:
Should this be in the no region?  it will just be ignored I suppose since it's not configured above.

Reply:
yea, we don't need that.  there could be more too.